### PR TITLE
August newsletter draft

### PR DESCRIPTION
**Pull request by bethaitman**
A little bit of editorialising in my article, but hopefully it's subtle enough.

### CODE REVIEW

File: docs/blog/newsletter-august-2020.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.. post:: August 02, 2020
+   :tags: newsletter
+
+#######################################
+Write the Docs Newsletter – August 2020
+#######################################
+
+Dear documentarians of the world - good morning, afternoon, or evening! It’s Beth here, writing to you on an absolutely sweltering day in Switzerland. Not going to lie, 36 degrees C in the home office is rather a challenge; on the plus side, at home a mid-afternoon siesta becomes a very viable option… 
+
+The `virtual Portland conference </conf/portland/2020>`__ is fast approaching, on 9-11 August. Tickets are `on the verge of selling out </conf/portland/2020/news/tickets-selling-out/>`__, so if you’ve been sitting on the fence about whether to go, get a move on! `Here’s the schedule </conf/portland/2020/news/schedule/>`__ if it’ll help you make up your mind.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2020.rst]: I think we're calling it online not virtual. Not that it makes a huge difference.

File: docs/blog/newsletter-august-2020.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+And if Portland isn’t in a convenient time zone for you, fear not! because the `Australia and India virtual conference </conf/australia/2020/>`__ has just announced both dates (3-4 December 2020) and a `call for proposals </conf/australia/2020/news/cfp-open/>`__. We cannot wait to hear from documentarians far and wide - get those talk proposals submitted by 31st August.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2020.rst]: > fear not! because 

guessing not intentional?

File: docs/blog/newsletter-august-2020.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Next month, the newsletter team is on our summer break - so enjoy this month’s articles, and I’ll look forward to popping back into your inbox again come October. Stay well!
+
+---------------------------------
+To git or not to git docs as code
+---------------------------------
+
+Everybody's asking about a docs-as-code workflow and toolchain these days - one recent question about adopting it for a team that's unfamiliar with Git sparked a useful discussion about this potentially rather painful learning curve. Here's the advice from folks who've brought teams up to speed with Git:
+
+* It's crucial to have at least one Git expert available to help troubleshoot, both as people are learning for the first time and as everyone on the team (no matter how proficient) will encounter new and unexpected issues.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2020.rst]: this needs rewording but I don't have enough coffee yet

> both as people are learning for the first time and as everyone on the team (no matter how proficient) will encounter new and unexpected issues

File: docs/blog/newsletter-august-2020.rst [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+* Consider GUI clients instead of the command line. Folks mentioned GitHub Desktop, TortoiseGit (Windows), Tower, Sourcetree, and GitKraken.
+* A lot of learning Git is about building confidence with an unfamiliar workflow and new underlying concepts. Once people get through the initial learning curve, stress levels tend to settle down.
+
+---------------------
+A tale of two careers
+---------------------
+
+Want to be a technical writer and work in a different field at the same time? Is it possible to juggle two careers without burning out? The reality is that technical writing requires a lot of time and effort; it’s not easy to balance this with other work, but it’s possible.
+
+Acknowledge and prepare for the difficulties. Accept that this balancing act is hard, and that’s okay. There are only so many hours in the day. If you can’t give 100% to both, then this isn’t the right path. It’s more realistic to turn one of your jobs into a side project. Having two careers implies serious intent to progress in both things simultaneously. This will take a lot of energy, devotion, and time spent working.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2020.rst]: This is super judgemental

> If you can’t give 100% to both, then this isn’t the right path.

File: docs/blog/newsletter-august-2020.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+If you’re determined to maintain the balancing act, occasionally switching between both jobs will give you a nice change of pace. In some cases, one job may inform the other or even provide you with a fresher perspective. Also, consider how you could merge responsibilities to create a hybrid role. They may be more compatible than you think.
+
+Reflect on how to create a long-term option that fulfills most of your desires. So long as you have the physical and mental time for the two, you’ll be happy with technical writing and the other career that balances it out.
+
+-------------------------------------------
+Content reuse vs hyperlinking
+-------------------------------------------
+
+Coming soon from Claire.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2020.rst]: Needs hyperlink. ;-) :japanese_goblin:

File: docs/blog/newsletter-august-2020.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+-------------------------------------
+What to do with tricky little details
+-------------------------------------
+
+In an ideal world, structuring documentation pages is easy. You cover the main concepts and the core tasks with clear, user-centered headings, and you’re done.
+
+But in messy reality, it can be tricky to fit all the relevant information into those headings. Where do you put non-standard uses, or fringe cases? Granular but important details that users won't find anywhere else? Questions that often come up in support?
+
+One option is to have a Frequently Asked Questions section at the bottom of the page. Readers are often familiar with FAQs as a format, and may even specifically request them. But FAQ can be a misnomer - for fringe cases, the questions are by definition not frequent. Besides, some folks feel that `FAQs are a poor content design pattern <https://gathercontent.com/blog/big-question-should-you-have-faqs-on-your-website>`__.
```
Comment (plaindocs) [inline docs/blog/newsletter-august-2020.rst:47]: There is an F for Fringe joke in here somewhere.

**Review by plaindocs — COMMENTED** [review]
I think more whimsical comments than my norm, hope that is ok. 
Awesome work from y'all.

File: docs/blog/newsletter-august-2020.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2020.rst]: my bad, fixed.

File: docs/blog/newsletter-august-2020.rst (hunk 2 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2020.rst]: deliberately part of the same sentence. But I know it's contentious!

File: docs/blog/newsletter-august-2020.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2020.rst]: Yeah, I wasn't quite sure how to put this one either. Have rephrased.

File: docs/blog/newsletter-august-2020.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2020.rst]: Good catch, I've softened it a bit.

File: docs/blog/newsletter-august-2020.rst (hunk 5 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2020.rst]: Still haven't heard back from Claire so removing.

File: docs/blog/newsletter-august-2020.rst (hunk 6 above)
Comment (bethaitman) [reply docs/blog/newsletter-august-2020.rst:47]: There is, but I'm too lazy to come up with it :P

**Pull request merged**